pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -309,6 +309,18 @@ self: super: {
   # > https://github.com/roelvandijk/numerals
   numerals = doJailbreak (dontCheck super.numerals);
 
+  # Bound on containers is too strict but jailbreak doesn't work with conditional flags
+  # https://github.com/NixOS/jailbreak-cabal/issues/24
+  containers-unicode-symbols = overrideCabal {
+    postPatch = ''
+      substituteInPlace containers-unicode-symbols.cabal \
+        --replace 'containers >= 0.5 && < 0.6.5' 'containers'
+    '';
+  } super.containers-unicode-symbols;
+
+  # Test file not included on hackage
+  numerals-base = dontCheck (doJailbreak super.numerals-base);
```

Comment:
Would you be able to file an issue (or even better, a PR) for this upstream, and link to it here in this comment?

Reply:
It's marked as deprecated in favor of `numerals` and not updated for 11 years so doubt there's a point